docs/how/how_buffer_format.rst

```diff
@@ -162,13 +162,13 @@ The buffer format supports annotations on declarations to specify special proper
 Struct definitions support the following annotations:
 
 * ``[[size(number)]]`` or ``[[byte_size(number)]]`` - Forces the struct to be padded up to a given size even if the contents don't require it.
-* ``[[single]]`` or ``[fixed]]`` - Forces the struct to be considered as a fixed SoA definition, even if in context the buffer viewer may default to AoS. See the below section for more details. Structs with this annotation **may not** be declared as a variable, and should instead be the implicit final struct in a definition.
+* ``[[single]]`` or ``[[fixed]]`` - Forces the struct to be considered as a fixed SoA definition, even if in context the buffer viewer may default to AoS. See the below section for more details. Structs with this annotation **may not** be declared as a variable, and should instead be the implicit final struct in a definition.
```

Comment:
It might be nice to copy over the ":ref:`the below section<aos-soa` that is used in the Variable section to replace the unlinked "the below section"

Reply:
I'd agree that this would be nice but it's a pre-existing issue not related to your fix, so if you'd rather I merge this as is then let me know and I will.

Reply:
I don't actually have a build environment set up for the documentation locally, so I'd rather not make that change myself just in case there's some nuance to the formatting I'm not aware of. But that does seem like a good idea.